symfony/sendgrid-mailer/4.4/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+    "env": {
+        "#1": "MAILER_DSN=sendgrid://KEY@default"
```

Comment:
Shouldn't it be `MAILER_DSN=sendgrid+smtp://KEY@default` to be consistent with 4.3 recipe?

Reply:
`sendgrid` is the equivalent of `sendgrid+smtp`. The default is now automatically the "best" transport possible.

Reply:
I got it, thank you for the clarification!